compiler/src/iree/compiler/Dialect/Flow/Transforms/RaiseSpecialOps.cpp

```diff
@@ -46,14 +144,22 @@ struct RaiseSpecialOpsPass : public RaiseSpecialOpsBase<RaiseSpecialOpsPass> {
         }
       }
     });
+
+    IRRewriter rewriter(getOperation()->getContext());
+
     for (std::pair<linalg::LinalgOp, Value> softmax : softmaxRoots) {
       linalg::LinalgOp op = softmax.first;
       Value src = softmax.second;
-      IRRewriter rewriter(op.getContext());
       rewriter.setInsertionPoint(softmax.first);
       rewriter.replaceOpWithNewOp<IREE::LinalgExt::SoftmaxOp>(
           op, src, op.getDpsInitOperand(0)->get(), op.getNumLoops() - 1);
     }
+
+    getOperation()->walk([&](linalg::LinalgOp op) {
+      IRRewriter rewriter(op.getContext());
```

Comment:
I think we can reuse the same `rewriter`, so we dont have to declare a new one in this walk. WDYT?